src/main.rs

```diff
@@ -39,6 +39,19 @@ struct Opts {
     )]
     ignore_file: String,
 
+    #[options(no_short, help = "Exclude files with this tag from the export")]
+    skip_tags: Vec<String>,
+
+    #[options(no_short, help = "Only include files with this tag in the export")]
```

Comment:
This is more concise. Would that sound better?

```suggestion
    #[options(no_short, help = "Export only files with this tag")]
```